src/infrastructure/database/repositories/games.repository.spec.js

```diff
@@ -451,4 +451,172 @@ describe("GamesRepository", function () {
       });
     });
   });
+
+  describe(".getTrendingGames returns an array of trending games based on the passed in arguments.", function () {
+    describe("If a specific time period is passed in", function () {
```

Comment:
Use process description instead of technical description for a unit under test. Here it would be: 
Describe: "If/When trending games for the last seven days are requested" <= change to similar
It: "Two games will be returned" <= change to something similar
It: ... <= the second it was correct
It: ... <= the third also